src/samples/.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) <2018> Intel Corporation
```

Comment:
<2020> since it's a newly created file.

Reply:
Thanks, fixed that and also updated the new peercall.js header